tests/main/ubuntu-core-snapd-run-from-snap/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Check that the snapd.run-from-snap helper works
+
+systems: [ubuntu-core-*]
+
+restore: |
+    # cleanup generated units (just in case stop does not do its job)
+    rm -f /run/systemd/system/snapd.*
+    # cleanup mount
+    umount /snap/snapd/current/usr/lib/snapd || true
+    # and ensure snapd is working (needed in restore)
+    systemctl daemon-reload
+    systemctl start snapd.service snapd.socket
+
+execute: |
+    echo "Install the snapd snap"
+    snap install --edge snapd
+
+    echo "Stop snapd from core"
+    systemctl stop snapd.service snapd.socket
+    systemctl status snapd|MATCH "Active: inactive"
+
+    echo "Ensure our local changes are in the snap"
+    mount -o bind /usr/lib/snapd /snap/snapd/current/usr/lib/snapd/
```

Comment:
Can you please move this to the prepare chunk. (along with the installation of snapd)